linkerd/app/integration/src/controller.rs

```diff
@@ -372,9 +372,7 @@ where
                 let _ = listening_tx.send(());
             }
 
-            #[allow(deprecated)] // linkerd/linkerd2#8733
-            let mut http = hyper::server::conn::Http::new().with_executor(TracingExecutor);
-            http.http2_only(true);
+            let http = hyper::server::conn::http2::Builder::new(TracingExecutor);
```

Comment:
an example of what we are doing here: replacing `conn::Http` values with a `conn::http2::Builder` or `conn::http1::Builder` as contextually appropriate.